### ISSUE TITLE
Bias in point-wise sep conv in BiFPN

### ISSUE THREAD

**Issue by dkoguciuk** [zylo117/Yet-Another-EfficientDet-Pytorch#11]
Hi @zylo117 ,

good job finding so many differences between other PyTorch repos and the official implementation. I've got one question about the BiFPN convolutions, i.e. here:

https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/blob/ff818b6d3a572dad9da72fbfc5a13314166ad65b/efficientdet/model.py#L74

You're using Separable Conv followed by the batch norm. My question is: why do you use bias in pointwise conv? Bias followed by batch norm is unnecessary, right?

**Comment by zylo117** [comment]
Yes, you're right. Theoretically BN's bias can work as last conv's bias, there is no need for another one. But it's too late for the change? It is there any painless solution that can somehow merge pointwise's bias into BN's bias?

I didn't add BN to SeparableConvBlock at first, but then I referred to the official implement, found out they apply BN on SeparableConvBlock of Regressor and Classifier, then I did the same. I didn't realize the there is a such redundant bias until you told me.

Well the only sliver lining is that, the official efficientdet also has unnecessary bias. Consider the result seems fine, so maybe just leave it be.
